stripe/src/main/java/com/stripe/android/StripeFireAndForgetRequestExecutor.kt

```diff
@@ -20,22 +27,24 @@ internal class StripeFireAndForgetRequestExecutor : FireAndForgetRequestExecutor
      */
     @VisibleForTesting
     @Throws(APIConnectionException::class, InvalidRequestException::class)
-    fun execute(request: StripeRequest): Int {
+    internal fun execute(request: StripeRequest): Int {
         // HttpURLConnection verifies SSL cert by default
         var conn: HttpURLConnection? = null
+        val responseCode: Int
         try {
             conn = connectionFactory.create(request)
 
             // required to trigger the request
-            val responseCode = conn.responseCode
+            responseCode = conn.responseCode
 
             closeConnection(conn, responseCode)
-            return responseCode
         } catch (e: IOException) {
             throw APIConnectionException.create(request.baseUrl, e)
         } finally {
             conn?.disconnect()
         }
+
+        return responseCode
```

Comment:
this change should be a no-op, returning from here or line 33 should work exactly the same